Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/model/Model.kt

```diff
@@ -33,7 +33,7 @@ open class Model(val modelInfo: ModelInfo, modelManager: ModelManager?) {
 
         for (t in index.values) {
             if (t is ClassType && t.label != null) {
-                classIndex[casify(t.label)] = t
+                classIndex[casify(t.label!!)] = t
```

Comment:
Are we sure we want to throw an NPE here? (!!)

Reply:
Ideally, no. I've been fixing the cases of these where it's reasonably straight forward to do so. In this case, the `label` property is mutable, meaning that it could change values between the null check (`t.label != null`) and the usage (`t.label!!`) if, for instance, the Model was shared between different threads. I looked at changing `label` to be immutable, but it requires a re-work of how the `ClassTypes` are initialized. I just decided it was a bigger change than I wanted do. I've _mostly_ kept the APIs and usages of the classes the same, only grabbing low-hanging fruit.